jitpack.yml

```diff
@@ -0,0 +1,9 @@
+jdk:
+  - openjdk8
+install:
+  - echo "Building non-compose artifacts"
+  - ./gradlew clean install -xtest
```

Comment:
Is not `publishToMavenLocal` enough for JitPack?

https://jitpack.io/docs/BUILDING

Reply:
Hmm. this was the recommendation from Jitpack when using a custom build script:
https://jitpack.io/docs/BUILDING/#custom-commands

Reply:
Does our Gradle config even has `install` command? Can you try to run it?

Reply:
it does. I have successfully run this script on a fork.
https://jitpack.io/#LachlanMcKee/RIBs/0.27.2.2

Reply:
![image](https://user-images.githubusercontent.com/1333155/109196522-b5dd2900-7793-11eb-8257-2b95682cb197.png)